charts/openstack-actions-runner/values.yaml

```diff
@@ -99,6 +99,7 @@ affinity: {}
 
 githubToken: ""
 githubOrganization: ""
+githubRepo: ""
```

Comment:
With this setting we will be forced to publish one instance for each repository.
Let's find a way were it's not the cases, it's to restrictive in the long terme.